src/frontend/js/login.js

```diff
@@ -149,11 +217,38 @@ div:has(#submit_btn) {
     this.password = inputEl.value;
   }
 
+  checkLogin(e) {
```

Comment:
il nome checkLogin non è evocativo di cosa fa la funzione

Reply:
cambiato in controlloTastoInvio()